jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/TargetImageParameters.java

```diff
@@ -60,10 +61,16 @@ public void setImage(String image) {
   @Input
   @Optional
   public Set<String> getTags() {
-    if (System.getProperty(PropertyNames.TO_TAGS) != null) {
-      return ImmutableSet.copyOf(
-          ConfigurationPropertyValidator.parseListProperty(
-              System.getProperty(PropertyNames.TO_TAGS)));
+    String property = System.getProperty(PropertyNames.TO_TAGS);
+    Set<String> tags =
```

Comment:
For readability: I think maybe `tags` is confusing with the class member `tags` (`this.tags`). Perhaps use something that differentiates it? Personally I don't think `this.tags` should be used very often outside constructors.